tests/integration/api/conftest.py

```diff
@@ -87,17 +87,59 @@ def test_metadata_entity() -> (
     return file
 
 
+@pytest.fixture
+def test_suspicious_metadata_entity() -> (
+    dict[str, list[str] | dict[str, str] | dict[str, str | dict[str, str]]]
```

Comment:
Do you really need this complicated of a type? Would this be better off as a `dict[str, Any]` as traditional JSON is usually represented? Or maybe don't do `.dict()` and just return the model as is?